System-Testing-Deployments-and-Upgrades/User-Authorization/index.rst

```diff
@@ -0,0 +1,162 @@
+
+.. |author| replace:: *Tiago Ribeiro*
+.. |contributors| replace:: *Wouter van Reeven*, *Rob Bovill*
+
+.. _User-Authorization:
+
+##################
+User Authorization
+##################
+
+The Rubin Observatory Control System has an authorization system designed to help manage and coordinate user interaction with the system.
+
+By default, the system is designed to be operating under autonomous CSC control.
+For example, during survey operations the Scheduler will compute the observing queue and feed them to the ScriptQueue, which then executes the SAL Scripts required to observe the target.
+
+At the same time, users may want to interact with the system to perform engineering or other tasks that require a finer level of interaction with the system, which is usually done from :ref:`nublado <Control-System-User-Interfaces-Nublado>`.
+Nevertheless, before they can interact with the system, they need to request authorization to directly command the CSCs.
+Users can also block CSCs from interacting with other CSCs if they want to guarantee nothing else in the system will interfere with their activities.
+
+.. _User-Authorization-Prerequisites:
+
+Prerequisites
+=============
+
+* Requesting authorization happens as part of interacting with the Observatory Control System.
+  Before you start make sure you have :ref:`announced the component usage <Operational-Tutorials-Annouce-Component-Usage>`.
+
+* The authorization system may require an observing specialist or other administrator approval (in LOVE).
+  Make sure you coordinate with the observing specialists to approve your request beforehand.
+
+* If you are an observing specialist or administrator, see :ref:`Managing-User-Authorization`.
+  
+
+.. _User-Authorization-Post-Condition:
+
+Post-Condition
+==============
+
+* User is authorized to control CSCs.
```

Comment:
or rejected, no?

Reply:
Well, I guess rejected would be an unwanted post condition.

Reply:
I think it's still a post-condition though so worth adding. 
I guess my high-level point here is that the docs are written in a way such that the outcome assumes success. 
Admittedly, having not seen nor used the auth service, it's hard to know if the feedback given to the user is enough to provide feedback.

Reply:
My reasoning here is that any expected failure scenario should be in Troubleshooting. I am afraid that if we add failures here, the Troubleshooting session will become redundant.

Reply:
ok that's fair, so just link that section from here.